packages/instrumentation-aws-sdk/package.json

```diff
@@ -32,15 +32,15 @@
         "url": "https://github.com/aspecto-io/opentelemetry-ext-js/issues"
     },
     "dependencies": {
-        "@opentelemetry/api": "^0.16.0",
-        "@opentelemetry/instrumentation": "^0.16.0",
-        "@opentelemetry/semantic-conventions": "^0.16.0",
-        "opentelemetry-propagation-utils": "^0.3.0"
+        "@opentelemetry/api": "^0.17.0",
+        "@opentelemetry/instrumentation": "^0.17.0",
+        "@opentelemetry/semantic-conventions": "^0.17.0",
+        "opentelemetry-propagation-utils": "^0.2.1"
```

Comment:
why did we downgrade the version of this package?

Reply:
merge mistake. thanks.